salt/metalk8s/addons/ui/config/metalk8s-shell-ui-config.yaml.j2

```diff
@@ -20,6 +20,7 @@ spec:
   logo:
     light: /brand/assets/logo-light.svg
     dark: /brand/assets/logo-dark.svg
+  favicon: /brand/assets/favicon-metalk8s.svg
```

Comment:
Same comment applies here I guess, would make more sense to set this as part of a coherent theme.